proposals/attach-to-property.md

```diff
@@ -0,0 +1,142 @@
+# Attach to delegate property object
```

Comment:
I think we should work some more on the title. Options off the top of my head:
- Attachment handlers for delegated properties
- Creation handlers for property delegates